cmd/agent/main.go

```diff
@@ -2,25 +2,18 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"log"
-	"net/http"
 	"os"
 
-	// Adds version information
-	_ "github.com/grafana/agent/pkg/build"
-
-	"github.com/grafana/agent/pkg/integrations"
-	"github.com/grafana/agent/pkg/loki"
-	"github.com/grafana/agent/pkg/tempo"
-
 	util_log "github.com/cortexproject/cortex/pkg/util/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/grafana/agent/pkg/config"
-	"github.com/grafana/agent/pkg/prom"
+
+	// Adds version information
+	_ "github.com/grafana/agent/pkg/build"
+
```

Comment:
You can remove the `_` imports from this file since they're already in `cmd/agent/agent.go` (GitHub won't let me comment on the other ones because you didn't change them 😞)